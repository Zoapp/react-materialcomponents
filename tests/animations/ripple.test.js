import React from "react";
import renderer from "react-test-renderer";

describe("animations/Ripple", () => {
  it("can have children", () => {
    const tree = renderer
      .create(<div>Interact with me !</div>, { ripple: true })
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("can be unbounded", () => {
    const tree = renderer
      .create(<div unbounded>Interact with me !</div>, { ripple: true })
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
