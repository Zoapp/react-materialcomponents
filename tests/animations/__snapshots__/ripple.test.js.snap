// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`animations/Ripple can be unbounded 1`] = `
<div
  unbounded={true}
>
  Interact with me !
</div>
`;

exports[`animations/Ripple can have children 1`] = `
<div>
  Interact with me !
</div>
`;
